backend/src/mirrors_qa_backend/extract.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any
+from urllib.parse import urlsplit
+
+import pycountry
+import requests
+from bs4 import BeautifulSoup, NavigableString
+from bs4.element import Tag
+
+from mirrors_qa_backend import logger, schemas
+from mirrors_qa_backend.exceptions import MirrorsExtractError
+from mirrors_qa_backend.settings import Settings
+
+
+def get_current_mirrors() -> list[schemas.Mirror]:
+    """
+    Returns list of current mirrors from the mrirors url.
```

Comment:
```suggestion
    Current mirrors from the mirrors url.
```

Reply:
Beside the typo, it is advised to omit “returns” wording and returned type as this is inferred from hints